library/SimplePie/Locator.php

```diff
@@ -153,7 +153,8 @@ public function is_feed($file, $check_html = false)
 			$sniffed = $sniffer->get_type();
 			$mime_types = array('application/rss+xml', 'application/rdf+xml',
 			                    'text/rdf', 'application/atom+xml', 'text/xml',
-			                    'application/xml', 'application/x-rss+xml');
+			                    'application/xml', 'application/x-rss+xml',
+					    'application/x-rss+xml, text/xml');
 			if ($check_html)
```

Comment:
BusinessWire apparently changed it's content-type header around 2019/02/27 to 'application/rss+xml, text/xml'.